VitruvioHost/Plugins/Vitruvio/Source/Vitruvio/Private/Util/AttributeConversion.cpp

```diff
@@ -312,6 +312,80 @@ TMap<FString, URuleAttribute*> ConvertAttributeMap(const AttributeMapUPtr& Attri
 	return UnrealAttributeMap;
 }
 
+void UpdateAttributeMap(TMap<FString, URuleAttribute*>& AttributeMapOut, const AttributeMapUPtr& AttributeMap, const RuleFileInfoUPtr& RuleInfo, UObject* const Outer)
+{
+	if (AttributeMapOut.Num() == 0)
+	{
+		AttributeMapOut = ConvertAttributeMap(AttributeMap, RuleInfo, Outer);

```

Comment:
Do we really need to call the old convert function here? Couldn't we just get rid of this if altogether (because the code below now handles both cases: attributes missing as well as updating). If so we could also get rid of the `ConvertAttributeMap` function (which is pretty similar to this function).

Reply:
Removed ConvertAttributeMap.